FeedAPIChallenge/Feed Feature/FeedImagesMapper.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by GD on 8/24/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feedImages: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
+
+	struct Item: Decodable {
+		public let image_id: UUID
+		public let image_desc: String?
+		public let image_loc: String?
+		public let image_url: URL
```

Comment:
`public` is not needed here since `Item` and `FeedImageMapper` are already `internal` and other modules won't be able to access (and shouldn't) the `Item` properties. You can remove the `public` access level.

```suggestion
		let image_id: UUID
		let image_desc: String?
		let image_loc: String?
		let image_url: URL
```